### ISSUE TITLE
SQL Error when adding anime to List

### ISSUE THREAD

**Issue by kuchienkz** [joshstar/AniList-API-Docs#105]
**Reproduction Steps:** Adding an anime to LIst (following the steps here: https://anilist-api.readthedocs.io/en/latest/lists.html#anime-list-create-entry-post-edit-entry-put)

**Reproduction Rate:** Tried 10 times, failed 10 times. So i would say it is Persistent.

**Summary of Bug:** Can’t add an anime to List. Always return this response:

**{“error”:{“status”:400,”messages”:[“SQL Error”]}}**

But when i add it from Browser, it works :|

This never happened before.

Thanks in Advance!

**Comment by danejur** [comment]
This is happening to me as well. However, even more strangely, Postman seems to work for me (while doing it in code does not and gives the 400).

**Comment by Kazakuri** [comment]
I can't reproduce it at all, can either of you provide a sample request?

For example:
```
POST https://anilist.co/api/animelist?id=21856&status=plan%20to%20watch
Authorization: Bearer <access_token>
Content-Type: application/json
```

That one works for me.

**Comment by danejur** [comment]
I have figured out what was wrong with my request, which may help @kuchienkz. (note: I have only been attempting these requests as updates, not creates).

While sending the request programatically, I realized that I was still attaching the "access_token" parameter, as this is loaded in as a "default parameter" for all requests made, even though these requests use a different auth method (in the header). Before the update, I guess this param was just ignored, but now that doesn't seem to be the case. Upon removing it, the update completes successfully.

I'd double check to make sure that you aren't accidentally including that parameter with your animelist and mangalist requests.
